### PR TITLE
Webumenia 1900 UI pre panel filtrov

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (link to Jira or GitHub issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] I have updated the [CHANGELOG](../CHANGELOG.md)
- [ ] I have requested at least 1 reviewer for this PR
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: resources/js/components/catalog-new/ShowMore.vue [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<template>
+    <div class="tw-min-w-max">
+        <button
+            @click="controller.toggleIsExtendedOpen"
```
Comment (eronisko) [inline resources/js/components/catalog-new/ShowMore.vue:4]: Do we really need to inject the controller here? I'd handle the state locally (in `data` in this component)

File: resources/views/frontend/catalog-new/index-new.blade.php [hunk 2]
```diff
@@ -2,13 +2,51 @@
...
 @section('content')
     <section class="tailwind-rules">
-        <filter-new :authors="{{ $authors }}">
-            <div class="tw-flex tw-space-x-3 tw-overflow-x-auto tw-bg-gray-200 tw-p-16">
+        <filter-new :authors="{{ $authors }}" v-slot="{ isExtendedOpen }">
+            <div class="tw-bg-gray-200 tw-p-16 md:tw-pb-0">
+                <div class="tw-flex tw-gap-x-3 tw-overflow-x-auto md:tw-flex-wrap">
+                    <div>
+                        <filter-new-custom-select :filter-name="'authors'"
+                            placeholder="Napíšte meno autora / autorky">
```
Comment (eronisko) [inline resources/views/frontend/catalog-new/index-new.blade.php:10]:
```suggestion
                        <filter-new-custom-select name="authors"
                            placeholder="Napíšte meno autora / autorky">
```

**Review by eronisko — CHANGES_REQUESTED** [review]
Looking good, just some minor suggestions

File: resources/js/components/catalog-new/Filter.vue [hunk 3]
```diff
@@ -20,6 +20,9 @@ export default {
         }
     },
     computed: {
+        selectedValues() {
+            return this.$route.query
+        },
```
Comment (eronisko) [inline resources/js/components/catalog-new/Filter.vue:25]:
```suggestion
        selectedValues() {
            return {
              ...this.$route.query,
              authors: typeof $this.route.query.authors === "string" ? [$this.route.query.authors] : $this.route.query.authors
            }
        },
```

**Review by eronisko — CHANGES_REQUESTED** [review]

File: resources/js/components/catalog-new/NewCustomCheckbox.vue [hunk 4]
```diff
@@ -0,0 +1,35 @@
+<template>
+    <label class="tw-flex tw-items-center tw-font-normal" :for="title">
```
[the remark this answers is not in the document]
Comment (FrantisekMichalSebestyen) [reply resources/js/components/catalog-new/NewCustomCheckbox.vue:2]: there is a css style coming from somewhere setting label it to bold 🤷

**Pull request merged**